mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/ProfileScreen.kt

```diff
@@ -198,3 +201,15 @@ private fun Modifier.borderBottom(
   val end = Offset(size.width, size.height)
   drawLine(color, start, end, width.toPx(), StrokeCap.Round)
 }
+
+/**
+ * A back button composable for the visited profile screen the that gets back to the previous screen
+ * when actioned
+ *
+ * @param onClick call back methode to action the back button
+ * @param modifier a Modifier for this composable
+ */
+@Composable
+private fun BackButton(onClick: () -> Unit, modifier: Modifier = Modifier) {
+  IconButton(modifier = modifier, onClick = onClick) { Icon(PawniesIcons.Close, "cancel") }

```

Comment:
The content description must be in `LocalizedStrings`.

Reply:
👍